Src/WitsmlExplorer.Api/Workers/TrimLogObjectWorker.cs

```diff
@@ -35,6 +35,16 @@ public override async Task<(WorkerResult, RefreshAction)> Execute(TrimLogDataJob
             Index currentEndIndex = Index.End(witsmlLog);
             Index newEndIndex = Index.End(witsmlLog, job.EndIndex);
 
+            if (newStartIndex > newEndIndex)
+            {
+                (newEndIndex, newStartIndex) = (newStartIndex, newEndIndex);
```

Comment:
This change is insufficient to allow trimming for decreasing logs, as the conditions on line 49 and line 71 never end up being true for decreasing logs.

Reply:
Fixed by specifing the order of logs by its Direction property, according to it the proper operation is selected